### ISSUE TITLE
Missing git release tags?

### ISSUE THREAD

**Issue by CheariX** [thundernest/quicktext#205]
Hi,

I observed that some Version released on [addons.thunderbird.net](https://addons.thunderbird.net/en-US/thunderbird/addon/quicktext/versions/) do not have corresponding [release tags](https://github.com/jobisoft/quicktext/releases).
E.g., v3.0, v3.2, v3.3, and v3.4 are missing.

This makes it a little hard to follow what changed from version to version.

Is there any reason for this or are the tags simply forgotten?

**Comment by SamuelPlentz** [comment]
Probably simply forgotten. You can keep track in the commit comments here:

- https://github.com/jobisoft/quicktext/commits/TB78

**Comment by jobisoft** [comment]
Creating tags/releases is not really part of my work flow. I do them from time to time.
